extensions/blocks/contact-form/editor.scss

```diff
@@ -39,6 +39,14 @@
 	width: 100%;
 }
 
+.jetpack-contact-form__thankyou-redirect-url input[type="text"] {
+	width: 100%;
+}
+
+.jetpack-contact-form__thankyou-redirect-url__suggestions {
+	width: 260px;
```

Comment:
This is a bit of a problem on smaller screens:
<img width="744" alt="Screenshot 2019-10-16 at 10 40 30" src="https://user-images.githubusercontent.com/275961/66907561-6e705580-f001-11e9-8296-3dc759cd0253.png">

Reply:
The default behaviour on small screens is nearly the same (302px wide, instead of 260px). `URLInput` is really only used in one or two places in Core, and not at all in the sidebar, so I'm not surprised it's a little weird. I'm happy to tweak it here, though.

Reply:
Actually this does that regardless of that line...
I think we can merge and address this separately.